### ISSUE TITLE
aws ubuntu ami hangs on launch

### ISSUE THREAD

**Issue by macintoshpie** [Parsl/parsl#1094]
The apt-get install command in the aws template.py results in a hang due to an interactive prompt:
```
whiptail--backtitlePackage configuration--titleConfiguring libssl1.1:amd64--output-fd11--defaultno--yesno--There are services installed on your system which need to be restarted 
when certain libraries, such as libpam, libc, and libssl, are upgraded. 
Since these restarts may cause interruptions of service for the system, 
you will normally be prompted on each upgrade for the list of services 
you wish to restart.  You can choose this option to avoid being 
prompted; instead, all necessary restarts will be done for you 
automatically so you can avoid being asked questions on each library 
upgrade.
```

## Replicate
Python: 3.6
Parsl: 0.8.0
```python
import parsl
from parsl.config import Config
from parsl.executors import HighThroughputExecutor
from parsl.providers import AWSProvider
from parsl.app.app import python_app

parsl.set_stream_logger()

parsl_config = Config(
  executors=[
    HighThroughputExecutor(
      label='htex_local',
      address=myip,
      cores_per_worker=1,
      max_workers=1,
      provider=AWSProvider(
        image_id='ami-0c55b159cbfafe1f0', # ubuntu 18.04
        instance_type='t2.micro',
        nodes_per_block=1,
        init_blocks=1,
        max_blocks=1,
        min_blocks=0,
        region='us-east-2',
        key_name='myKeyName'
      ),
    )
  ],
  strategy=None,
)

parsl.load(parsl_config)

@python_app
def hello ():
  return 'Hello World!'

print(hello().result())
```

**Comment by benclifford** [comment]
Yesterday on slack we talked about adding a line

```
export DEBIAN_FRONTEND=noninteractive
```

near the start of `template.py`.

Did that work?

**Comment by benclifford** [comment]
fixed by #1098
